### PR TITLE
added readme.md

### PR DESCRIPTION
**Pull request by Sekhar-Kumar-Dash**
## Related Issue
  - Info about Issue or bug

Closes: #338 

#### Describe the changes you've made
A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Please let us know if any test cases are added

Please describe the tests(if any). Provide instructions how its affecting the coverage.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.

## Checklist:

- [x] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.

## Screenshots (if any)

|        Original         |          Updated           |
|:-----------------------:|:--------------------------:|
| **original screenshot** | <b>updated screenshot </b> |

### CODE REVIEW

File: readme1.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+Swarm Testing: Swarm testing involves generating a large number of tests that cover a wide range of inputs and conditions. This can be done using a tool like Dredd, which can generate tests based on the API schema.
+Pros:
+
+Covers a wide range of inputs and conditions.
+Can be automated.
+Cons:
+
+Can be time-consuming and resource-intensive.
+May not cover all test cases.
+
```
Comment (charankamarapu) [inline readme1.md:40]: Good work with exploring different techniques for generating test cases from api schema . But I guess there is no that good technique in the solutions that you mentioned above . Because every technique has a con that it may not cover all cases or edge cases . Majorly that is the problem we are trying to solve over here with this task. As It is a tiresome process for user to write edge cases we are introducing this so that user will not have to do much.

File: readme1.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+.................. here is  how we can take the API schema from the user and load it into Keploy............................
+
+Accepting API schema: Keploy should have a feature that allows users to upload or provide an API schema, which describes the endpoints, methods, parameters, and expected responses of the API. The API schema can be provided in various formats such as OpenAPI specification (formerly Swagger), RAML, WADL, etc. Keploy should be able to accept these formats.
+
+Validating API schema: Once the API schema is provided, Keploy should validate it against the respective schema specification to ensure its correctness. This step is essential to avoid any issues while generating tests.
+
+Parsing API schema: Keploy should parse the API schema and extract relevant information such as endpoints, methods, parameters, and expected responses. This step is crucial for generating tests that cover all possible scenarios.
```
Comment (charankamarapu) [inline readme1.md:47]: Please mention specifications how will it extract form schema .

File: readme1.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+Storing API schema: After parsing the API schema, Keploy should store the extracted information in a structured manner, which can be used for generating tests and other related purposes.
+
+Integrating with other tools: Keploy can integrate with other tools such as API mocking tools, API testing tools, API documentation tools, etc., to provide a comprehensive API testing solution. For example, Keploy can use an API mocking tool to simulate the API and generate test cases automatically.
+
+Providing a user interface: Keploy should provide a user interface where users can upload the API schema, view the parsed information, and generate test cases. The user interface should be intuitive and easy to use.
```
Comment (charankamarapu) [inline readme1.md:53]: Keploy server mostly CLI based . We also prefer this task to be mostly CLI based

File: readme1.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+.................. here is  how we can take the API schema from the user and load it into Keploy............................
+
+Accepting API schema: Keploy should have a feature that allows users to upload or provide an API schema, which describes the endpoints, methods, parameters, and expected responses of the API. The API schema can be provided in various formats such as OpenAPI specification (formerly Swagger), RAML, WADL, etc. Keploy should be able to accept these formats.
+
+Validating API schema: Once the API schema is provided, Keploy should validate it against the respective schema specification to ensure its correctness. This step is essential to avoid any issues while generating tests.
+
+Parsing API schema: Keploy should parse the API schema and extract relevant information such as endpoints, methods, parameters, and expected responses. This step is crucial for generating tests that cover all possible scenarios.
+
+Storing API schema: After parsing the API schema, Keploy should store the extracted information in a structured manner, which can be used for generating tests and other related purposes.
```
Comment (charankamarapu) [inline readme1.md:49]: Please mention specifications what is the structure you want it to be stored in and why..?

File: readme1.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+Providing a user interface: Keploy should provide a user interface where users can upload the API schema, view the parsed information, and generate test cases. The user interface should be intuitive and easy to use.
+
+
+
+
+
+
+........here is How  we can convert that schema and produce automated tests which has almost every possible testcase........
+To convert the schema and produce automated tests, we can use code generation tools. These tools take the schema definition and generate code in a specific programming language. For example, we can use tools like OpenAPI Generator, Swagger Codegen, or graphql-codegen to generate code for REST or GraphQL APIs.
```
Comment (charankamarapu) [inline readme1.md:61]: generate code ..? what does it mean ? does it mean testcases..?
